### ISSUE TITLE
Add test variable button is broken

### ISSUE THREAD

**Issue by RoboErikG** [google/blockly#3185]
**Describe the bug**

If you drag out one of the variable validators in the test blocks and then try to add test variables you get errors in the console.

**To Reproduce**

1. Open the playground and switch to test blocks
2. Drag out one of the variable validator test blocks.
3. Click the 'add test variable' button in the flyout

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Stack Traces**

```
variable_map.js:178 Uncaught Error: Variable "1a" is already in use and its id is "]T:Ws0tgqg3)z-H)MZ*]" which conflicts with the passed in id, "1A".
    at Blockly.VariableMap.createVariable (variable_map.js:178)
    at Blockly.WorkspaceSvg.Blockly.Workspace.createVariable (workspace.js:522)
    at Blockly.WorkspaceSvg.createVariable (workspace_svg.js:1326)
    at addVariables (playground.html?dir=…&renderer=geras:244)
    at Blockly.FlyoutButton.onMouseUp_ (flyout_button.js:260)
    at SVGGElement.wrapFunc (blockly.js:473)
```

**Additional context**

**Comment by BeksOmega** [comment]
This is known - I thought I added a comment about it but I must not have.

It might be easiest to just add a note saying, "be sure to add the variables before dragging the validator blocks out".

**Comment by RoboErikG** [comment]
Yeah, low priority to fix but might be good for someone learning how variables work to dig into.

**Comment by BeksOmega** [comment]
> Wouldn't it be enough to just not create the variable?
> e.g. remove line 244 in the following snippet and the error will be fixed.
> https://github.com/google/blockly/blob/17af987de0de726528033f24a5916418b82acb54/tests/playground.html#L243-L250
> 
> 
> As far as I can tell there is no way the variable can not exist, because if a block is dragged out, the variable `a1` will be created anyway.

Sounds reasonable to me, I'd give it a shot!

**Comment by alschmiedt** [comment]
Fixed in #3346
